native/acter/src/api/verification.rs

```diff
@@ -1014,7 +1015,10 @@ impl SessionManager {
                             session: info.session.clone(),
                         });
                         let auth_data = AuthData::Password(pass_data);
-                        client.delete_devices(&devices, Some(auth_data)).await?;
```

Comment:
these changes seem unrelated to the description of this PR. Or I don't understand their relationship. please explain.

Reply:
I just wanted to use `delete_device()` not `delete_devices()`, because argument of this api was changed from `Vec<String>` into `String`.
I found `ruma` provides `delete_device` api and used it.

Reply:
Raised [another PR](https://github.com/acterglobal/a3/pull/2017) and moved this there.